tests/integration/session_spec.lua

```diff
@@ -392,6 +392,10 @@ describe("session", function()
 		jump(1)
 	end)
 	it("Deleting nested snippet only removes it.", function()
+		ls_helpers.session_setup_luasnip({
+			setup_extend = { exit_roots = false },
+			hl_choiceNode = true,
```

Comment:
Can `hl_choiceNode` be omitted?

Reply:
During my tests, it was not possible. I think this is because `before_each` function call adds `hl_choice = true`, then calling `ls_helpers.session_setup_luasnip` here discards this `hl_choice = true` setting.

However, if you want I can change snippet to work without `hl_choiceNode`.

Reply:
Ohhh, yeah ofc... you could also theoretically pass `exit_roots = false` in `before_each`, all test in `session` should work with that.
Actually, I think I'd prefer that, seems a bit cleaner :D